demos/interactive_face_detection_demo/README.md

```diff
@@ -1,5 +1,7 @@
 # Interactive Face Detection C++ Demo
 
+![](./interactive_face_detection.gif)
```

Comment:
I think it's better to use all other nets the demo supports like emotions to show all capabilities.

Reply:
done.